CRM/Mosaico/Page/Index.php

```diff
@@ -25,6 +25,23 @@ function run() {
           ts('PHP extension Fileinfo not loaded or enabled')
         );
       }
+      if (empty($config->extensionsURL)) {
```

Comment:
@deepak-srivastava i have removed these lines again in https://github.com/veda-consulting/uk.co.vedaconsulting.mosaico/pull/57